app/models/concerns/vacancy_application_detail_validations.rb

```diff
@@ -2,9 +2,11 @@ module VacancyApplicationDetailValidations
   extend ActiveSupport::Concern
 
   included do
+    validates :application_link, url: true, if: :application_link?
+
     validates :contact_email, presence: true
     validates :contact_email, format: { with: /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\z/i }, if: :contact_email?
 
-    validates :application_link, url: true, if: :application_link?
+    validates :contact_number, format: { with: /\A\+?(?:\d\s?){10,12}\z/ }, if: :contact_number?
```

Comment:
I'm curious what 10 and 12 refer to here

Reply:
Between 10 and 12 digits allowed